features/support/env.rb

```diff
@@ -1,5 +1,6 @@
+require 'aruba'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.